src/quest_manager/views.py

```diff
@@ -248,16 +248,20 @@ class QuestCopy(QuestCreate):
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
 
-        # by default, set the quest this was copied from as the new_quest_prerequisite
-        # If this is changed in the form it will be overwritten in form_valid() from QuestFormViewMixin
-        copied_quest = get_object_or_404(Quest, pk=self.kwargs["quest_id"])
+        # Use a queryset that includes archived quests for staff
+        qs = Quest.objects.all()
```

Comment:
by default this won't include archived quests (because the get_queryset ddefault)  Maybe read up on get_queryset method override that we did , so you don't have to change everything uneccessarily, and can just get the queryset you want exactly.